.drone.yml

```diff
@@ -425,14 +425,14 @@ steps:
     pull: if-not-exists
     image: plugins/slack
     settings:
-      channel: sas-build
+      channel: sas-hof-build-notify
       failure: ignore
       icon_url: http://readme.drone.io/0.5/logo_dark.svg
       icon.url: http://readme.drone.io/0.5/logo_dark.svg
       template: "CRON Job {{build.deployTo}} of Change of Address has {{build.status}} - <{{build.link}}|#{{build.number}}> {{#success build.status}}\n  :thumbsup: :thumbsup: :thumbsup:\n{{else}}\n  :x: :x: :x:\n{{/success}} Author: {{build.author}}\n\nDuration: {{since job.started}}\n\nJob: <{{build.link}}|#{{build.number}}>\n\nCommit: {{build.commit}}\n"
```

Comment:
```
template: >
            :x: CRON Job `tear_down_pr_envs` for Change of Address (COA) has failed.
            Repo `{{ repo.name }}` branch <https://github.com/{{ repo.owner }}/{{ repo.name }}/commits/{{ build.branch }}|{{build.branch}}> for commit <https://github.com/{{ repo.owner }}/{{ repo.name }}/commit/{{ build.commit }}|{{ truncate build.commit 8 }}>
```

Reply:
Thank you very much @dk4g . Much appreciated . I have updated in the code